jina/clients/request/asyncio.py

```diff
@@ -46,15 +46,17 @@ async def request_generator(
             with ImportExtensions(required=True):
                 import aiostream
 
-            async for batch in aiostream.stream.chunks(data, request_size):
-                yield _new_data_request_from_batch(
-                    _kwargs=kwargs,
-                    batch=batch,
-                    data_type=data_type,
-                    endpoint=exec_endpoint,
-                    target=target_executor,
-                    parameters=parameters,
-                )
+            # breakpoint()
+            async with aiter(aiostream.stream.chunks(data, request_size)) as streamer:
```

Comment:
@alaeddine-13 The `aiter` function is available only from python version 3.10.

Reply:
then we only do aiter and async with if python version is >3.10